.github/workflows/build-sample-apps.yml

```diff
@@ -66,11 +68,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Install tools used in workflow
-      run: |
-        brew install xcbeautify    
-        brew install sd
-
```

Comment:
why don't we need these anymore? I remember i previously saw your comment on one of the PRs where you mention "You expected X to be available on Linux but that was not the case and had to install it manually"

Reply:
This is removing a duplicate entry. If you look further down in the workflow file you see that this code is copy/pasted. I made a mistake and added this code twice.